src/pkg/config/config.go

```diff
@@ -1,11 +1,95 @@
 package config
 
 import (
+	"github.com/otterize/otterize-cli/src/pkg/utils/must"
+	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
 )
 
+const OtterizeConfigDirName = ".otterize"
+
+func OtterizeConfigDirPath() (string, error) {
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(home, OtterizeConfigDirName), nil
+}
+
+const OtterizeConfigFileName = "config"
+const EnvPrefix = "OTTERIZE"
+
+const ApiClientIdKey = "client-id"
+const ApiClientSecretKey = "client-secret"
```

Comment:
what is our usecase for these? are we sure they're still viable and need migrating to the public cli?

Reply:
I didn't want to change the fundamental features in the tools CLI repo. this is here in case the user wants to use the CLI as an integration for some reason